operations/vouchers.md

```diff
@@ -322,62 +364,74 @@ Updates information about the specified vouchers. Note this operation supports [
-| `AssignedRateIds` | [Array string update value](_objects.md#array-of-strings-update-value), max 5 items | optional | Unique identifiers of [Rates](rates.md#rate) \(or `null` should it not be updated\). |
-| `OccupiableIntervalStartUtc` | [String update value](_objects.md#string-update-value) | optional | Start of the time interval, expressed as the timestamp for the start of the first [time unit](services.md#time-unit), in UTC timezone ISO 8601 format \(or `null` if the start time should not be updated\). |
-| `OccupiableIntervalEndUtc` | [String update value](_objects.md#string-update-value) | optional | End of the time interval, expressed as the timestamp for the start of the last [time unit](services.md#time-unit), in UTC timezone ISO 8601 format \(or `null` if the end time should not be updated\). |
-| `ExternalIdentifier` | [String update value](_objects.md#string-update-value) | optional, max 255 characters | Identifier of the voucher from external system \(or `null` if the identifier should not be updated\). |
+| `VoucherId` | string | required | Unique identifier of the Voucher. |
+| `Name` | [String update value](_objects.md#string-update-value) | optional, max length 128 characters | Internal name of the voucher (or `null` if the name should not be updated). |
+| `Type` | [VoucherTypeUpdateValue](vouchers.md#vouchertypeupdatevalue) | optional | Type of the voucher (or `null` if the type should not be updated). |
+| `CompanyId` | [String update value](_objects.md#string-update-value) | optional | Unique identifier of Company (Company or Travel Agency) the voucher is related to. This is required for Type of `PartnerCompany` or `TravelAgency`. Use `null` if Company should not be updated. |
+| `AssignedRateIds` | [GuidIEnumerableUpdateValue](vouchers.md#guidienumerableupdatevalue) | optional, max length 5 characters | Unique identifiers of Rates (or `null` should it not be updated). |
+| `OccupiableIntervalStartUtc` | [String update value](_objects.md#string-update-value) | optional | Start of the time interval, expressed as the timestamp for the start of the first time unit, in UTC timezone ISO 8601 format (or `null` if the start time should not be updated). |
+| `OccupiableIntervalEndUtc` | [String update value](_objects.md#string-update-value) | optional | End of the time interval, expressed as the timestamp for the start of the last time unit, in UTC timezone ISO 8601 format (or `null` if the end time should not be updated). |
+| `ExternalIdentifier` | [String update value](_objects.md#string-update-value) | optional, max length 255 characters | Identifier of the voucher from external system (or `null` if the identifier should not be updated). |
+
+#### VoucherTypeUpdateValue
```

Comment:
❗ ToDo: will need some help with fixing this one

Reply:
I think this could be linked to [string update value](https://mews-systems.gitbook.io/connector-api/operations/_objects#string-update-value) and in the description you'd specify it's a voucher type. But I have a suspicion it could be the first time in the migration we're dealing with enum in update value. 🤔

Reply:
[fixed here](https://github.com/MewsSystems/gitbook-connector-api/pull/622/commits/b937a7b49c1e488296f275fd7a912fc71a2fc97c) as suggested but hardcoded enum values in description smell fishy 👃